packages/ssh-agent/ssh-agent.0.1.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+author: "Reynir Björnsson <reynir@reynir.dk>"
+dev-repo: "https://github.com/reynir/ocaml-ssh-agent.git"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+license: "BSD-2-clause"
+build: ["jbuilder" "build" "-p" name]
+depends: [
+  "ocamlfind" {build}
```

Comment:
since you're using `jbuilder`, do you actually need ocamlfind?

Reply:
Maybe not. I think it's just boilerplate I copied from *somewhere* :-)